charts/core/src/Chart/Chart.spec.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import '@testing-library/jest-dom/extend-expect';
+
+import { ChartProvider } from '../ChartContext';
+
+import { Chart } from './Chart';
+
+jest.mock('../ChartContext', () => ({
+  ChartProvider: jest.fn(({ children }) => <div>{children}</div>),
+}));
+
+jest.mock('./hooks/useChart', () => ({
+  useChart: jest.fn(() => ({
+    chartOptions: {},
+    updateChartOptions: jest.fn(),
+    addSeries: jest.fn(),
+    removeSeries: jest.fn(),
+  })),
+}));
+
+jest.mock('@leafygreen-ui/leafygreen-provider', () => ({
+  useDarkMode: jest.fn(() => ({ theme: 'light' })),
+}));
+
+describe('lg-charts/core/Chart', () => {
```

Comment:
Wasn't sure what else could/should be tested for this component without falling too heavily into implementation detail testing. I leaned towards relying more on visual regression and chromatic tests here, but open to suggestions.